client/src/pages/queries/components/history-editor-tab.tsx

```diff
@@ -220,55 +236,152 @@ function Tab() {
           </Button>
         </div>
 
-        <div className="flex flex-col flex-1" ref={measureContainer}>
-          <VerticalPane
-            initialHeight={480}
-            maxHeight={containerBounds.height}
-            maxConstraint={containerBounds.height - 160}
-            buffer={80}
-            render={({
-              paneHeight,
-              isFullScreen,
-              dragHandle,
-              toggleFullScreen,
-            }) => (
-              <>
-                {!isFullScreen ? (
+        <div
+          className={clsx(
+            'flex flex-1',
+            splitOrientation === SplitOrientation.VERTICAL ? '' : 'flex-col',
```

Comment:
Always return the positive result at first. It can be done as `splitOrientation === SplitOrientation.HORIZONTAL ? 'flex-col' : undefined`